### PR TITLE
Proposal ONIX technique for Charts, diagrams, and formulas

### PR DESCRIPTION
**Pull request by gregoriopellegrino**
Thanks to Chris here is the proposal for the ONIX technique for "Charts, diagrams, and formulas"

### CODE REVIEW

File: UX-Guide-Metadata/draft/techniques/onix-metadata/index.html [hunk 1]
```diff
@@ -233,8 +233,8 @@ <h4>Variables setup</h4>
...
+					<li><b>LET</b> <var>real_text</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "10" or ProductContentType = "10"]</code>.</li>
+					<li><b>LET</b> <var>non_textual_content_still</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>onix</var>, <code class="xpath">/ONIXMessage/Product/DescriptiveDetail[PrimaryContentType = "07" or PrimaryContentType = "18" or PrimaryContentType = "19" or PrimaryContentType = "12"  or PrimaryContentType = "49" or PrimaryContentType = "20" or ProductContentType = "07" or ProductContentType = "18" or ProductContentType = "19" or ProductContentType = "12"  or ProductContentType = "49" or ProductContentType = "20"]</code>.</li>
```
Comment (clapierre) [inline UX-Guide-Metadata/draft/techniques/onix-metadata/index.html:237]: I thought we were going to change _still to something more descriptive. like _still_image ?

Comment (gautierchomel) [reply UX-Guide-Metadata/draft/techniques/onix-metadata/index.html:237]: [Preview](https://rawcdn.githack.com/w3c/publ-a11y/1abe0943cbc2bbfe3d35c88b90f0125f383336dc/UX-Guide-Metadata/draft/techniques/onix-metadata/index.html)

**Comment by gregoriopellegrino** [comment]
I've updated the pull request based on Charles' suggestion and on comments during last call (check only for metadata declaring that accessible STEM content is present, without checking whether the publisher declares the presence of STEM content in the ebook).

[Preview](https://rawcdn.githack.com/w3c/publ-a11y/ddd5baf2cfb971cdf50a31fff21ef235113b0241/UX-Guide-Metadata/draft/techniques/onix-metadata/index.html#charts-diagrams-and-formulas)

I would proceed with the merge.

**Review by clapierre — APPROVED** [review]
Reads much better thanks Gregorio.

**Comment by madeleinerothberg** [comment]
Looks good overall. Minor suggestion: plural or singular? We have a mix. Charts, images, diagrams, but formula. Having even one in the book makes the adaptation necessary but maybe it reads better if all are plural?

**Comment by chrisONIX** [comment]
It does look good.
If we go for plural of formula - have to decide if use formulae or formulas.
I have used formulae in the suggested text but formulas is equally good.




Christopher Saynor
EDItEUR
United House, North  Road
London N7 9DP
UK
Tel: +44 20 7503 6418

The information contained in this e-mail is confidential and may be privileged. It is intended for the addressee only. If you are not the intended recipient, please inform the sender and delete this e-mail immediately. The contents of this e-mail must not be disclosed or copied without the sender's consent. We cannot accept any responsibility for viruses, so please scan all attachments. The statements and opinions expressed in this message are those of the author and do not necessarily reflect those of the company.

EDItEUR Limited is a company limited by guarantee, registered in England no 2994705. Registered Office: United House, North Road, London N7 9DP, UK. Website: https://www.editeur.org

For Thema codes https://ns.editeur.org/thema/
For ONIX codes  https://ns.editeur.org/onix/



On 3 Apr 2024, at 15:42, Madeleine Rothberg ***@***.***> wrote:



Looks good overall. Minor suggestion: plural or singular? We have a mix. Charts, images, diagrams, but formula. Having even one in the book makes the adaptation necessary but maybe it reads better if all are plural?

**Review by gautierchomel — APPROVED** [review]

**Comment by GeorgeKerscher** [comment]
This looks good. I will merge.

**Comment by gregoriopellegrino** [comment]
> Yes, I was wondering about formulae vs formulas. Quick check online suggests that "ae" is used in scientific contexts and is also more common in British English. We are currently using "color" not "colour" which suggests we are using American spelling rules. Who is our supreme authority on copyedit rules? (Joking, but we do need a way to make decisions.)

If we make a decision, I will proceed with the changes.

**Comment by GeorgeKerscher** [comment]
My vote would be for US English. Of course, that is where I live Do we change the lange attribute to en-us? It seems reasonable and will be important for the localizations.

Best
George

**Comment by chrisONIX** [comment]
We use US-english for ONIX by default - even if we are based in the UK - I would say use what is understandable to the majority of users

Christopher Saynor
EDItEUR
United House, North  Road
London N7 9DP
UK
Tel: +44 20 7503 6418

The information contained in this e-mail is confidential and may be privileged. It is intended for the addressee only. If you are not the intended recipient, please inform the sender and delete this e-mail immediately. The contents of this e-mail must not be disclosed or copied without the sender's consent. We cannot accept any responsibility for viruses, so please scan all attachments. The statements and opinions expressed in this message are those of the author and do not necessarily reflect those of the company.

EDItEUR Limited is a company limited by guarantee, registered in England no 2994705. Registered Office: United House, North Road, London N7 9DP, UK. Website: https://www.editeur.org

For Thema codes https://ns.editeur.org/thema/
For ONIX codes  https://ns.editeur.org/onix/



On 4 Apr 2024, at 14:41, George ***@***.***> wrote:



My vote would be for US English. Of course, that is where I live Do we change the lange attribute to en-us? It seems reasonable and will be important for the localizations.

Best
George

**Comment by gregoriopellegrino** [comment]
Sorry, since I am neither British English nor American English, could you please point me to the changes that need to be made?

Thank you 😀

**Comment by chrisONIX** [comment]
The consensus is to use the US English version of the plural of formula - that is formulas.

So if the word ‘formulae’ is present please replace this with ‘formulas’


Thank you



Christopher Saynor
EDItEUR
United House, North  Road
London N7 9DP
UK
Tel: +44 20 7503 6418

The information contained in this e-mail is confidential and may be privileged. It is intended for the addressee only. If you are not the intended recipient, please inform the sender and delete this e-mail immediately. The contents of this e-mail must not be disclosed or copied without the sender's consent. We cannot accept any responsibility for viruses, so please scan all attachments. The statements and opinions expressed in this message are those of the author and do not necessarily reflect those of the company.

EDItEUR Limited is a company limited by guarantee, registered in England no 2994705. Registered Office: United House, North Road, London N7 9DP, UK. Website: https://www.editeur.org

For Thema codes https://ns.editeur.org/thema/
For ONIX codes  https://ns.editeur.org/onix/



On 8 Apr 2024, at 07:39, Gregorio Pellegirno ***@***.***> wrote:



Sorry, since I am neither British English nor American English, could you please point me to the changes that need to be made?

Thank you 😀

**Pull request merged**